Documentation/RELEASING.md

```diff
@@ -48,9 +48,45 @@ When doing a release:
    everything based on the PR descriptions and _semver_ tags in the repo. Just read
    though was was generate to see if any tweaks are needed.
 
-   When everything is good, click on _Publish release_.
+   **Important:** Save this as a **draft** release first (do not publish yet).
+
+4. Generate protoc artifact bundle
+
+   Once you have created the draft release, trigger the "Upload protoc artifactbundle" 
+   workflow from the [Actions tab](https://github.com/apple/swift-protobuf/actions/workflows/prerelease_protoc_artifactbundle.yml).
+   
+   This workflow will:
+   - Fetch the latest stable protoc release from protocolbuffers/protobuf
+   - Create a Swift Package Manager compatible artifact bundle
+   - Upload it to your draft release
+   
+   Wait for the workflow to complete successfully before proceeding.
+
+5. Update Package.swift with new artifact bundle
+
+   Create a pull request that updates the `Package.swift` file to reference the new 
+   artifact bundle. You'll need to update two things:
+   
+   - **URL**: Change to point to your new release tag
+   - **Checksum**: Download the artifact bundle and calculate its SHA256 hash
+   
+   Example update:
+   ```swift
+   .binaryTarget(
```

Comment:
I guess this will exist in the Package.swift once we do our first release with this in place, but for now it doesn't yet exist?

Once we have this, could we then use it for all our Test targets and not check in the generated files for development?  We'd still need them in the runtime and plugin library, but would reduce the need for the tests?  (and I guess Reference/ still needs them all)  And the plugin/compile tests that use the plugin could also make use of it?

Reply:
Yes correct. I have some of those changes ready for a subsequent PR :)